src/provider/model/ConfigStore.ts

```diff
@@ -8,23 +8,27 @@ import {AsyncInit} from '../controller/AsyncInit';
 @injectable()
 export class ConfigStore extends AsyncInit {
     private _store: Store<ConfigurationObject>;
-    private _loader: Loader<ConfigurationObject>;
+    private _loader!: Loader<ConfigurationObject>;
 
     constructor() {
         super();
         this._store = new Store(require('../../../gen/provider/config/defaults.json'));
-        this._loader = new Loader(this._store, 'fdc3');
+        if (global.hasOwnProperty('fin')) {
```

Comment:
Rather than having these checks for `fin`, should we have a different `ConfigStore` that gets used in tests?

Reply:
Removed!